app/ide-desktop/lib/performance-logger/package.js

```diff
@@ -0,0 +1,6 @@
+let config = {
+    name: 'performance-logger',
```

Comment:
I really don't like the name "performance logger". This framework is not a logger. I think we should use the naming described in the tech doc. If that naming is bad, let's create discussion about it in the tech doc repo first.

Reply:
Changed.